inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -31,16 +31,18 @@ Moreover, the hardware and interaction are configured with the usual general par
 
 ### Example
 
+To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
+
 ```python
  import inductiva
 
  # Initialize the scenario
  scenario = inductiva.molecules.scenarios.ProteinSolvation(
-     protein_pdb = "alanine.pdb"
+     protein_pdb = "alanine.pdb",
      temperature = 300)
 
  # Run a simulation
- task = scenario.simulate(simulation_time_ns = 20)
+ task = scenario.simulate(simulation_time_ns = 0.5)
 
  # Get the simulation output on your local machine.
  output = task.get_output()
```

Comment:
The images below will be outdated / not matching alanine... let's try to use alanine images, or alternatively provide as an example the protein that we rendered there...